app/src/main/AndroidManifest.xml

```diff
@@ -9,13 +9,16 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.ParkingSystemKotlin">
-        <activity android:name=".MainActivity">
+        <activity android:name=".activities.MainActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".activities.ParkingSpaceReservationActivity">
+
+        </activity>
```

Comment:
remove this close tag and just finish the line 19 with /> please

Reply:
Done!